benchmarks/benchmark_serving.py

```diff
@@ -765,6 +804,23 @@ def main(args: argparse.Namespace):
         "{backend}-{args.request_rate}qps-{base_model_id}-{current_dt}.json"
         " format.",
     )
+    parser.add_argument(
+        "--percentile-metrics",
```

Comment:
nit: could we have this as simply `--metrics`? I think having `--percentile-metrics` and `--metric-percentiles` together might be confusing to users.

Reply:
From the doc string, it's clear what they are. In fact, it's kind of confusing even if we rename it to metrics because there are metrics always computed.